src/components/Heading.tsx

```diff
@@ -30,27 +30,27 @@ export const Heading = React.forwardRef<HTMLHeadingElement, HeadingProps>((props
             textIndent: '-.05em',
           },
           1: {
-            fontSize: 5,
+            fontSize: 4,
```

Comment:
Better to switch to sass file and generate style in it

Reply:
CSS-in-JS is a wide debate and I don't think that it should be handle in this particular PR since the whole project is handle this way. Feel free to open an issue.